.pre-commit-config.yaml

```diff
@@ -41,7 +41,7 @@ repos:
       - id: mypy
         additional_dependencies: [types-ipaddress, types-paramiko, types-python-dateutil, types-pytz, types-PyYAML, types-requests]
         exclude: "vulture_allowlist.py"
-        args: [--ignore-missing-imports]
+        args: [--ignore-missing-imports, --check-untyped-defs]
```

Comment:
What's the default behavior? Why is it set now? The commit message doesn't explain it

Reply:
Mypy was not checking untyped definitions: 
<img width="850" alt="image" src="https://user-images.githubusercontent.com/15820737/224293402-3870beba-b403-4643-a595-865377b7040e.png">

Reply:
Do you mean that the default is not checking the body of a method that does not have type annotations? What does this change has to do with this PR?